antora.yml

```diff
@@ -8,3 +8,9 @@ nav:
 
 asciidoc:
   attributes:
+    minimum-php-version: 7.3
```

Comment:
10.7 supports PHP 7.2 - I guess you will do a separate PR to update the specific values?

Reply:
Yes, will be done in a next PR

Reply:
yes, see my question in the 10.8 backport